### DISCUSSION TITLE
Specify default asset preview mode

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9502] [Ideas]
> Created by: Jacob Graf (jacob@graftechnology.com) on 2015/03/10 20:53:55 +0000
> Votes at time of UserVoice import: 1

-----------------------------------

Would love to be able to set an asset source to default to "grid" mode instead of list view. Most image asset fields are way more intuitive when browsing in grid view instead of list view.

**Comment by johanneslamers** [comment]
Yes please
